### PR TITLE
Add vue-mce

### PR DESCRIPTION
**Pull request by Eazymov**
Hi!
Please add [vue-mce](https://github.com/Eazymov/vue-mce), it is the best tinymce component for vuejs
Category: UI Components / Form / Rich Text Editing
- The project has documentation
- The documentation is in English.
- The project is active and maintained.
- The project accepts contributions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1046,6 +1046,7 @@ Tooltips / popovers
 
 #### Rich Text Editing
 
+ - [vue-mce](https://github.com/Eazymov/vue-mce) - tinymce editor component for VueJS.
```
Comment (kazupon) [inline README.md]: > Always add your items to the end of a list. To be fair, the order is first-come-first-serve.

https://github.com/vuejs/awesome-vue/blob/master/CONTRIBUTING.md

`CONTRIBUTING.md` have some rule in the above.
Could you fix the it please?

File: awesome_vue_with_repo_info.md [hunk 2]
```diff
@@ -996,6 +996,12 @@
     </a>
    </li>
    <li>
+    <a href="https://github.com/Eazymov/vue-mce">
```
Comment (kazupon) [inline awesome_vue_with_repo_info.md]: You don't need to add to `awesome_vue_with_repo_info.md`, due to not use.

**Comment by Eazymov** [comment]
Fixed

**Comment by kazupon** [comment]
Thanks! :)

**Pull request merged**